### DISCUSSION TITLE
Sizing image served by Zipline

### DISCUSSION THREAD

**Discussion by heathcurtiss** [diced/zipline#580] [Questions]
I'm a newb, but I'm stoked to have Zipline up and running on my Unraid server.  On my Mac I installed homebrew, Flameshot, jq, and xsel, and then created an application and script in Automator that does a nice job of automating the upload to Zipline.  I've copied my script below in case it helps anyone else as I have a couple hours into it (and I'm totally open to improvements).  My question is, when the image is served by Zipline it's set to display full screen (example [here](https://zip.curtiss.io/u/pJFRP1.png)).  It would be better if it showed at the original size/pixels.  Is there a way to make Zipline serve the image at a more reasonable size?  I've confirmed that when I upload it directly to Zipline and grab the link there, it's displayed properly(example [here](https://zip.curtiss.io/u/j0Szwu.png)).  Why does the html differ so much when uploading with my script?

Script below:

```
#!/bin/bash

export PATH="/Applications/flameshot.app/Contents/MacOS:$PATH"
export PATH="/opt/homebrew/bin:$PATH"

# Set the Pictures folder path and create a unique filename
PICTURES_PATH="$HOME/Pictures"
TIMESTAMP=$(date +"%Y%m%d_%H%M%S")
OUR_FILE="${PICTURES_PATH}/screenshot_${TIMESTAMP}.png"

# Take the screenshot
flameshot gui --path "${OUR_FILE}"

# Check if aborted
if [ ! -s "${OUR_FILE}" ]; then
    display notification "Screenshot was aborted, so we don't upload!" with title "Zipline upload script"
    rm -f "${OUR_FILE}"  # Remove empty file if created
    exit 1
fi

# Upload
curl -H "authorization: GETTHISFROMZIPLINE" https://DOMAIN/api/upload \
     -F file="@${OUR_FILE}"  \
     -H "Content-Type: multipart/form-data" \
     -H "Format: random" \
     -H "Embed: true" \
  | jq -r '.files[0]' \
  | tr -d '\n' \
  | pbcopy

# Optionally, remove the file after upload
# rm -f "${OUR_FILE}"

```

**Comment by diced** [comment]
Wasn't able to access your images, could you upload them directly to github? (screenshot your entire browser window on the image zipline is serving). As far as I remember, i'm pretty sure it will show the image at original size until the window becomes too small - in that case it will try to scale it up or down. I'm also unsure if I understand the problem here correctly, so it would be helpful if you can share those screenshots soon :)
